rotkehlchen/tests/exchanges/test_kraken.py

```diff
@@ -21,17 +21,22 @@ def test_name():
 
 
 def test_coverage_of_kraken_balances(kraken):
-    # Since 05/08/2019 Kraken removed all delisted assets from their public API
-    # query except for BSV. No idea why or why this incosistency.
     got_assets = set(kraken.api_query('Assets').keys())
-    expected_assets = (set(KRAKEN_TO_WORLD.keys()) - set(KRAKEN_DELISTED)).union({'BSV'})
+    expected_assets = (set(KRAKEN_TO_WORLD.keys()) - set(KRAKEN_DELISTED))
     # Ignore the staking assets from the got assets
     got_assets.remove('XTZ.S')
     got_assets.remove('DOT.S')
     got_assets.remove('ATOM.S')
     got_assets.remove('EUR.M')
     got_assets.remove('USD.M')
     got_assets.remove('XBT.M')
+    got_assets.remove('KSM.S')
+    got_assets.remove('EUR.HOLD')
+    got_assets.remove('FLOW.S')
+    got_assets.remove('FLOWH.S')
+    # Ignore the following assets as well
```

Comment:
When i said ignore, I meant to ignore it as part of the warning, because I don't know how to deal with them.

I can't find a reference to FLOW anywhere yet, so not sure how to properly support it.

Perhaps it's fine to leave it like that, but we have to remember to undo this if we actually ever support it.